### ISSUE TITLE
Allow citing draft OGC documents (as references and citations)

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/metanorma-ogc#120]
Originally from @ogcscotts: https://github.com/metanorma/metanorma-ogc/issues/99#issuecomment-604494457

This ticket relates to "non-published OGC documents" when cited or referred to.

1. If a non-published OGC document is cited within document content it should be displayed as:
`{document-number} (draft)`.

e.g. "OGC 03-105r1 (draft)"

2. In the Bibliography, the non-published OGC document should be shown as:

```
[6]  Open Geospatial Consortium: OGC 11-145 (Draft), Cyberarchitecture for Geosciences White Paper. Open Geospatial Consortium (2014).
```

**Comment by opoudjis** [comment]
If an OGC document is not published, it presumably will not be retrieved from relaton-ogc. But we cannot assume that every OGC document not found in relaton-ogc is a draft OGC document!

This ticket is requiring not only that a "draft" label be found on unpublished documents, for citation, but that documents not retrieved from relaton-ogc are well defined with a year and publisher. This presumably means that they are well-formatted using AsciiBib? If not, I'd be string searching for "(draft)" in the formatted reference.

**Comment by opoudjis** [comment]
So: 

* "(draft)" will be appended to the document identifier in citations, if that document is OGC, and it has a status, and that status/stage is other than "published", "deprecated", or "retired".
* Status can be entered in AsciiBib bibliographies
* A draft status will be surmised if "(draft)" is seen in a manually formatted OGC bibliographic entry (for which there is no retrieval from relaton-ogc). If it is, we will arbitrarily assign it the status "draft", given that all the documented OGC statuses seem quite advanced.
* The strange expectation of a date for a work in draft will be met if there is something that looks like a year in parentheses in the preformatted title. If AsciiBib is used, the expectation is that that year appear in `date[@type = 'circulated']`, since drafts by definition do not have dates of publication.

@ronaldtse Please confirm these assumptions; not proceeding without them.

**Comment by opoudjis** [comment]
Will rely exclusively on relaton-ogc to indicate draft status. Therefore, ticket on hold pending https://github.com/relaton/relaton-ogc/issues/5

**Comment by opoudjis** [comment]
Document status in `bibitem/status/stage` is considered draft if it is populated, and other than published, retired, or deprecated

**Comment by opoudjis** [comment]
I have taken care of the erefs in Presentation XML, but @Intelligent2013 you will need to implement this for references rendering in parallel.
